samza-core/src/main/java/org/apache/samza/zk/ZkKeyBuilder.java

```diff
@@ -45,6 +45,7 @@
 
   static final String PROCESSORS_PATH = "processors";
   static final String PROCESSOR_ID_PREFIX = "processor-";
```

Comment:
Yes. I think this was probably due to some merge conflict. @sborya let's not prefix the node name with "processor-" .